descartes_planner/test/dense_planner.cpp

```diff
@@ -37,7 +39,7 @@ class ThreeDOFRobot: public descartes_trajectory_test::CartesianRobot
 {
 public:
   ThreeDOFRobot():
-    descartes_trajectory_test::CartesianRobot(0,0,3)
+    descartes_trajectory_test::CartesianRobot(0,0)
```

Comment:
I'm going to entirely entirely scrap this robot and use the existing cartesian test robot.